lms/services/mailchimp.py

```diff
@@ -40,6 +42,64 @@ def __init__(self, db, api_key):
         self.db = db
         self.mailchimp_client = mailchimp_transactional.Client(api_key)
 
+    def send(  # pylint:disable=too-many-arguments
```

Comment:
This `send()` method is largely a duplicate of the `send_template()` method below. A following PR will delete `send_template()` once the message queue has been drained of `send_template()` tasks.